### PR TITLE
Howto for local transactions

### PR DESCRIPTION
**Pull request by gszol**
I have not yet figured out how to run this, but it should be mostly OK.
It might be a while before I have time for it again, and wanted to share it for review.

### CODE REVIEW

File: howtos/pubsub/how_to_use_local_transactions.py [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        try:
+            number_of_queues = 5
+            messages_per_queue = 3
+            messaging_service = MessagingService.builder().from_properties(boot.broker_properties()).build()
+            messaging_service.connect()
+            print(f'Message service is connected? {messaging_service.is_connected}')
+            topics = []
+            queues = []
+            for i in range(number_of_queues):
+                topic_name = constants.TOPIC_ENDPOINT_DEFAULT + str(i)
```
Comment (TamimiGitHub) [inline howtos/pubsub/how_to_use_local_transactions.py]: Can we use `constanst.TOPIC_ENDPOINT` instead to advocate for best practices with topics using hierarchical levels. thanks!

File: howtos/pubsub/how_to_use_local_transactions.py [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+                                  access_type="exclusive",
+                                  egress_enabled=True)
+                semp.add_topic_to_queue(topic_name=topic_name,
+                                        queue_name=queue_name,
+                                        msg_vpn_name=broker_props[service_properties.VPN_NAME])
+                queue = Queue.durable_exclusive_queue(queue_name)
+                queues.append(queue)
+
+                # Publish the expected number of messages to the queues
+                HowtoForLocalTransactions.publish_to_topics(messaging_service, topics, messages_per_queue)
```
Comment (TamimiGitHub) [inline howtos/pubsub/how_to_use_local_transactions.py]: For the samples sake, can we assign this to a `messages_delivered_to_topics` var and just print it

**Comment by TamimiGitHub** [comment]
Thanks @gszol for your PR! I added two comments, but overall looks good to me :) I will merge after you make any more changes

File: howtos/pubsub/how_to_use_local_transactions.py (hunk 1 above)
Comment (gszol) [reply howtos/pubsub/how_to_use_local_transactions.py]: Done.
It's value is "purchase/tickets/local_transaction_sample", but it's centrally configurable, so I'm not worried.

File: howtos/pubsub/how_to_use_local_transactions.py (hunk 2 above)
Comment (gszol) [reply howtos/pubsub/how_to_use_local_transactions.py]: Done.
I'm also printing out the received messages while at it.

**Comment by gszol** [comment]
Aside from making the two changes I also managed to run the sample and fix some bugs, so now it should run for customers too :-) 
BTW the README links to https://solace.com/products/event-broker/software/getting-started/ which exposes plaintext SEMP on port 8080, so I had to change that in my local solbroker_properties.json (the line "semp.hostname": "https://localhost:1943")

**Comment by gszol** [comment]
Could this be merged please so I can close off the corresponding JIRA tickets? (I don't have write permission on this repo)
TIA!

**Pull request merged**